ui/packages/consul-ui/app/templates/dc/acls/auth-methods/show/namespace-rules.hbs

```diff
@@ -0,0 +1,21 @@
+
+<div class="tab-section">
+{{#if (gt item.NamespaceRules.length 0)}}
+  <p>A set of rules that can control which namespace tokens created via this auth method will be created within. Unlike binding rules, the first matching namespace rule wins.</p>
+  <Consul::AuthMethod::NamespaceTable @items={{item.NamespaceRules}} />
```

Comment:
Here's another thing that is a little difficult to give reasons/rules for naming things. 

Ideally this one would be `Consul::AuthMethod::NamespaceRule::List`. List here doesn't necessarily mean 'this is formatted using `li`s' more that this is multiple of them rather than a single one. Then if we ever need to change how we markup the `NamespaceRule::List` (so not use tables anymore), we don't have to rename all the areas in the code where we use the component.